game/script-topics.rpy

```diff
@@ -159,6 +159,163 @@ label talk_print_keymap:
+    
+    return
+
+# This topic allows us to set the watched item list from file
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._topic_database,
+            label="talk_load_watched_items_from_disk",
+            unlocked=True,
+            prompt="Can you load the watched item list from disk?",
+            conditional="config.console",
+            category=["Debug (Watch)"],
+            player_says=True,
+            location="classroom"
+        ),
+        topic_group=TOPIC_TYPE_NORMAL
+    )
+
+label talk_load_watched_items_from_disk:
+    n "No problemo,{w=0.1} [player]!{w=0.2} Uno momento..."
+    n "..."
+    $ load_outcome = jn_debug.load_tracked_watch_items_from_disk()
+
+    if load_outcome == jn_debug.LOAD_FROM_DISK_NEW_FILE_CREATED:
+        n "...Huh.{w=0.2} Hey,{w=0.1} [player]..."
+        n "It looks like the file I needed didn't exist,{w=0.1} so I went ahead and created it for you."
+        $ import os
+        $ file_created_directory = os.getcwd()
+        n "It's called {i}watch_items.txt{/i},{w=0.1} and you should be able to find it in the {i}debug{/i} folder under {i}[file_created_directory]{/i}."
+        n "Remember though -{w=0.1} only one statement per line in the file,{w=0.1} 'kay?{w=0.2} Ehehe."
+
+    elif load_outcome == jn_debug.LOAD_FROM_DISK_SUCCESS:
+        n "And...{w=0.3} presto -{w=0.1} job done!"
+
+    else:
+        n "Uhmm...{w=0.3} [player]?{w=0.2} It looks like something went wrong."
+        n "Are you sure the file is actually there, and isn't blank or anything dumb like that?"
+        n "Just let me know when you've checked it out,{w=0.1} 'kay?"
+        n "Thanks,{w=0.1} [player]~!"
+
+    return
+
+# This topic allows us to reset the watched item list to its basic configuration
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._topic_database,
+            label="talk_reset_watched_items",
+            unlocked=True,
+            prompt="Can you reset the watched item list?",
+            conditional="config.console",
+            category=["Debug (Watch)"],
+            player_says=True,
+            location="classroom"
+        ),
+        topic_group=TOPIC_TYPE_NORMAL
+    )
+
+label talk_reset_watched_items:
+    n "Oh?{w=0.2} You want to reset the watched item list?"
+    if len(persistent.jn_debug_tracked_watch_items) > 10:
+        n "Uhmm...{w=0.3} [player]?{w=0.2} It looks like you're watching a lot of stuff..."
+        menu:
+            n "Are you sure you want to reset the list?"
+            "Yes, please reset it.":
+                pass
+            "No, not just yet.":
+                n "Oh...{w=0.3} well,{w=0.1} alright.{w=0.2} Just let me know whenever then,{w=0.1} 'kay?"
+                return
+
+    n "Okaaay!{w=0.2} Just give me a second..."
+    n "..."
+    $ jn_debug.reset_tracked_watch_items()
+    n "And...{w=0.3} gone -{w=0.1} it should be back to basics now!{w=0.2} Ehehe."
+    return
```

Comment:
We should add dev stuff to a folder for them and their own files, so they won't end up in production builds